ch01.asciidoc

```diff
@@ -86,6 +86,11 @@ Mining for bitcoin::
 
 Each of these stories is based on the real people and real industries currently using bitcoin to create new markets, new industries, and innovative solutions to global economic issues.((("", startref="GSuses01")))
 
+[TIP]
+====
+Have you recently checked the bitcoin exchange rate against your desired currency? As explained in <<bitcoin_price>>, the price of bitcoin is moving! Some examples in this book reference real-life _past_ transactions that have taken place, which allow readers (or anyone!) to search for them on the blockchain using online tools. In later chapters, interacting with this data can deepen your understanding of how Bitcoin operates. Having said this, today's exchange rate might suggest that the characters in these stories have highly irregular motivations. At the end of this chapter, as a fun thought experiment, think about how generous of a friend Joe (one of Alice's friends, whom you will soon meet) would have to be if he had made the same agreement with Alice today. In the next chapter, you might also notice that Alice _really_ loves coffee!
```

Comment:
And also change:

> Some examples in this book reference real-life _past_ transactions that have taken place, which allow readers (or anyone!) to search for them on the blockchain using online tools. In later chapters, interacting with this data can deepen your understanding of how Bitcoin operates. Having said this, today's exchange rate might suggest that the characters in these stories have highly irregular motivations. At the end of this chapter, as a fun thought experiment, think about how generous of a friend Joe (one of Alice's friends, whom you will soon meet) would have to be if he had made the same agreement with Alice today. In the next chapter, you might also notice that Alice _really_ loves coffee!

To:

> Some examples in this book reference real-life _past_ transactions that have taken place. Think about how generous of a friend Joe (one of Alice's friends, whom you will soon meet) would have to be if he had made the same agreement with Alice today. In the next chapter, you might also notice that Alice _really_ loves coffee!

Reply:
@krupawan5618 

Thanks for the quick change. Just wanted to tag this one again in case you missed it since I wrote it at the same time you updated the paragraph.

Reply:
👍 I like it, nice and succinct :)